App.tsx

```diff
@@ -1,8 +1,10 @@
 import * as Sentry from '@sentry/react-native';
-import React from 'react';
+import React, { Component } from 'react';
 import { I18nextProvider } from 'react-i18next';
-import { View, StyleSheet, LogBox } from 'react-native';
+import { View, StyleSheet, LogBox, Platform } from 'react-native';
 import codePush from 'react-native-code-push';
+import Config from 'react-native-config';
```

Comment:
hmm can we use global config  from folder config in src ?

Reply:
fixed